powermon/__init__.py

```diff
@@ -142,89 +148,53 @@ def main():
         print("# default location ./powermon.yaml")
         print(yaml.dump(config))
         return
+    # debug config
+    log.info("config: %s", config)
 
-    # debug dump config
-    log.info(f"config: {config}")
+    # build mqtt broker object (optional)
+    mqtt_broker = MqttBroker(config=config.get("mqttbroker", {}))
```

Comment:
think this is the best model, init the object with the config and handle errors, exceptions in the object class